README.md

```diff
@@ -25,25 +25,46 @@ Installation
 ------------
 IRBEM requires a [Fortran compiler](https://fortran-lang.org/learn/os_setup/install_gfortran), and can be installed on modern Windows, Linux, and Mac computers.
 
-- The quick build procedure on Linux with gfortran:
+## Linux
+The quick build procedure on Linux with gfortran:
 ```bash
     git clone https://github.com/PRBEM/IRBEM.git
     cd IRBEM
     make OS=linux64 ENV=gfortran64 all
     make OS=linux64 ENV=gfortran64 install
 ```
 
-- The quick build procedure on Mac OSX with gfortran:
+## Mac OSX
+The quick build procedure on Mac OSX with gfortran:
 ```bash
     git clone https://github.com/PRBEM/IRBEM.git
     cd IRBEM
     make OS=osx64 ENV=gfortran64 all
     make OS=osx64 ENV=gfortran64 install
 ```
 
-- The quick build procedure for Windows will be created soon. In the meanwhile, one workaround is to use the Windows Subsystem for Linux (WSL).
+## Windows
+The build procedure for 64-bit Windows with gfortran is involved, mainly due to the Fortran compiler. 
 
-See the `README.install` file for more details.
+1. Download and install [MSYS2](https://www.msys2.org/) and follow the [Fortran installation steps](https://www.msys2.org/#:~:text=and%20what%20for.-,Installation,-Download%20the%20installer) (summarized below). 
+   1. Run MSYS2 MSYS terminal from the start menu and update the MSYS2 packages via ```pacman -Syu``` (pacman is its package manager). Close the terminal after packages are updated. 
+   2. Run MSYS2 MSYS from the start menu again and update the rest of the packages using ```pacman -Syu```.
```

Comment:
They do appear redundant, but the [MSYS2](https://www.msys2.org/) docs includes both steps. Their rationale is "Update the package database and base packages. Unless your setup file is very recent, it will take two steps." 

Instead of asking IRBEM users to go to MSYS2 docs and follow their steps, we summarized the main commands here.